rmw_zenoh_cpp/src/detail/event.cpp

```diff
@@ -183,7 +167,7 @@ void EventsManager::add_new_event(
 }
 
 ///=============================================================================
-void EventsManager::attach_event_condition(
+bool EventsManager::queue_has_data_and_attach_condition_if_not(
```

Comment:
Suggest to standardize to `queue_is_empty` instead of `queue_has_data`.

```suggestion
bool EventsManager::queue_is_empty_and_attach_condition_if_not(
```